charts/hono/README.md

```diff
@@ -548,19 +542,20 @@ helm install --dependency-update -n hono --set messagingNetworkTypes[0]=kafka --
 The parameters enable the deployment of an example Kafka cluster, disable the deployment of the AMQP 1.0 messaging network 
 and configure adapters and services to use Kafka based messaging.
 
-To use the service type `NodePort` instead of `LoadBalancer`, the following parameters must be added: 
+To use the service type `NodePort` instead of `LoadBalancer`, the following parameters must be added:
 `--set useLoadBalancer=false --set kafka.externalAccess.service.type=NodePort`.
 
 ### Using a production grade Kafka cluster
 
 If Kafka based messaging is enabled by adding `kafka` to `messagingNetworkTypes`, the Kafka clients need to
-be configured with connection information for a Kafka cluster. The Helm chart can deploy an example Kafka cluster. 
+be configured with connection information for a Kafka cluster. The Helm chart can deploy an example Kafka cluster.
 This is enabled by setting `kafkaMessagingClusterExample.enabled` to `true`. With this setting the chart
 deploys a Kafka cluster consisting of a single broker and a single Zookeeper instance and configures the 
 protocol adapters to connect to the example cluster.
 
-In a production environment, though, usage of the example Kafka cluster is strongly discouraged as it does not provide 
-any redundancy.
+However, usage of the example Kafka cluster in a production environment is
+strongly discouraged as it is a single node cluster only which does not
```

Comment:
I would put a comma after "only", or use "as it is only a single node cluster which does not provide any redundancy".